pkg/api/validation/validation.go

```diff
@@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD *api.GCEPersistentDisk) errs.ValidationErrorLi
 	return allErrs
 }
 
+func ValidatePersistentVolume(persistentvolume *api.PersistentVolume) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	return allErrs
```

Comment:
I think volume plugins need to participate in validation at the master eventually @thockin

----- Original Message -----
> > @@ -119,6 +119,16 @@ func validateGCEPersistentDisk(PD
> > *api.GCEPersistentDisk) errs.ValidationErrorLi
> >  	return allErrs
> >  }
> >  
> > +func ValidatePersistentVolume(persistentvolume *api.PersistentVolume)
> > errs.ValidationErrorList {
> > +	allErrs := errs.ValidationErrorList{}
> > +	return allErrs
> 
> If the goal is to support generic persistent disks, how to validate the
> parameters?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/3318/files#r23336056